spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
If access mode is part of `VolumeCapability` as well as `VolumeInfo` then `ValidateVolumeCapabilities` seems confusing because it has both data types, and each would have an `AccessMode`.

If access mode is NOT part of `VolumeInfo` what prevents the SP from satisfying the request with a mode equal to or better than what was requested? If the SP upgrades the access mode in a CreateVolume request, does the CO really care as long as it gets the minimum behavior it needs for the workload it wants to run?